test/node/api-versions.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+const test = require('tap');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application');
+
+const interfacesV1 = {
+  calculator: {
+    answer(connection, callback) {
+      callback(null, 42);
+    },
+  }
+};
+
+const interfacesV2 = {
+  calculator: {
+    answer(connection, callback) {
+      callback(null, 24);
+    },
+  }
+};
+
+const interfacesVLatest = {
```

Comment:
No, It was made to check specific behavior (when app has no version it's latest version). If every app has a version then the biggest version will be the latest, see https://github.com/metarhia/jstp/pull/231/files/66c24103eef081bdb56c42696a400182502634f5#diff-13dd866a30aedfb1fab7c3d9da9446ceR162.